### DISCUSSION TITLE
Geotiff as a map

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#18608] [Maps Related]
I see that terrain maps (3D) use tiff file for rendering.
Is it possible to use geotiff map then?
I put the map to  geotiff folder but it seems that it doesn't work.
Do I do something in wrong way or geotiff maps are not supported?

**Comment by vshcherb** [comment]
Geotiff is already used as Heightmap, well I think there is nothing wrong to support Geotiff  in future as raster layer.

**Reply by mariush444** [reply]
Thank you. 
What is a difference between geotiff highmap and standard.
Highmap shows some shadow/shape of a mountains.
May I modify my geotiff somehow to be visible by highmap?

**Reply by vshcherb** [reply]
Nope, there is an issue to support gdal (https://github.com/osmandapp/OsmAnd/issues/18625) so I think we will add support for geotif as well

**Reply by DL179GL** [reply]
The terrain maps are ± 40GB about the same of HGT DEM ones so with (I think) the same definition for **ground** details.
On the GEBCO web site (https://www.gebco.net/data_and_products/gridded_bathymetry_data/) are available some files (less detailed) in TIF format that included the undersea values (for unknown reason these files are not currently readble on Osmand - the same trying to translate the other formats via QGIS). As for the efforts done by Osmand in "nautical", I think to read the numeric depht value could be great

**Comment by wolfbert** [comment]
I don't know what you are trying to achieve and it's been a while, but just in case ...

The official 1:50.000 topo map of Austria is available in GEOTIFF. I've used [MAPC2MAPC](http://www.the-thorns.org.uk/mapping/) with GDAL installed (as mentioned in [OSMAND documentation](https://osmand.net/docs/technical/map-creation/create-offline-maps-yourself#mapc2mapc)) to produce an sqlite database and it works as an overlay or underlay map. I'm happy to share the steps involved. 
Be aware of the storage requirements, in my case a 50x50 km square for zoom level 15 requires about 115 MB.

**Reply by mariush444** [reply]
The problem is the size. Geotiff is much much smaller than sqlitedb.

**Reply by wolfbert** [reply]
I'm not an expert on this. In my case, sqlitedb is about 20% of the tif size. Still a lot, but then its a topo map and compression goes only so far.
